cns/NetworkContainerContract.go

```diff
@@ -63,12 +68,14 @@ type CreateNetworkContainerRequest struct {
 	Routes                     []Route
 	AllowHostToNCCommunication bool
 	AllowNCToHostCommunication bool
+	EndpointPolicies           []NetworkContainerRequestPolicies
 }
 
-// ConfigureContainerNetworkingRequest - specifies request to attach/detach container to network.
```

Comment:
why did you remove ConfigureContainerNetworkingRequest struct?

Reply:
Thanks for catching this, corrected it.